wallet/subnet/primary/common/options.go

```diff
@@ -35,7 +35,7 @@ type Options struct {
 	baseFee *big.Int
 
 	minIssuanceTimeSet bool
-	minIssuanceTime    uint64
+	minIssuanceTime    int64
```

Comment:
Because we still treat all times as `uint64` for outputs - I feel like this should still be a `uint64`.